cts/cts-cli.in

```diff
@@ -1856,27 +1856,28 @@ for t in $tests; do
     grep -e '^\* \(Passed\|Failed\)' "$TMPFILE"
 done
 
+function print_or_remove_file {
+  eval TMPFILE="\$TMPFILE_$t"
+  if [[ $(diff -w $test_home/cli/regression.$t.exp "$TMPFILE") ]]; then
+    rm -f "$TMPFILE"
+  else
+    echo "    $TMPFILE"
+  fi
+}
```

Comment:
This looks good, but it will be more readable if you take the test name as a function argument rather than treat it as a global variable, like:

    print_or_remove_file "$t"

That still treats $test_home as a global, but it already is elsewhere (and makes more sense as a global)